### PR TITLE
install and set up guide update

### PR DESCRIPTION
**Pull request by priyankatuteja**
\<insert pull request description here\>

-----

# Checklist

Please go through each entry in the below checklist and mark an 'X' if that condition has been met. Every entry should be marked with an 'X' to be get the Pull Request approved.

- [x] All `import`s are in the first cell? First block of imports are standard libraries, second block are 3rd party libraries, third block are all `arcgis` imports? Note that in some cases, for samples, it is a good idea to keep the imports next to where they are used, particularly for uncommonly used features that we want to highlight.
- [x] All `GIS` object instantiations are one of the following?
    - `gis = GIS()`
    - `gis = GIS('https://www.arcgis.com', 'arcgis_python', 'P@ssword123')`
    - `gis = GIS(profile="your_online_profile")`
    - `gis = GIS('https://pythonapi.playground.esri.com/portal', 'arcgis_python', 'amazing_arcgis_123')`
    - `gis = GIS(profile="your_enterprise_portal")`
- [x] If this notebook requires setup or teardown, did you add the appropriate code to `./misc/setup.py` and/or `./misc/teardown.py`?
- [x] If this notebook references any portal items that need to be staged on AGOL/Python API playground, did you coordinate with a Python API team member to stage the item the correct way with the api\_data\_owner user?
- [x] Code refactored & split out across multiple cells, useful comments?
- [x] Consistent voice/tense/narrative style? Thoroughly checked for typos?
- [x] All images used like `` instead of ``? All map widgets contain a static image preview? (Call `mapview_inst.take_screenshot()` to do so)
- [x] All file paths are constructed in an OS-agnostic fashion with `os.path.join()`? (Instead of `r"\foo\bar"`, `os.path.join(os.path.sep, "foo", "bar")`, etc.)
- [ ] **IF YOU WANT THIS SAMPLE TO BE DISPLAYED ON THE DEVELOPERS.ARCGIS.COM WEBSITE**, ping @ DavidJVitale so he can add it to the list for the next deploy

### CODE REVIEW

**Comment by priyankatuteja** [comment]
@jyaistMap Adding a do not merge label as I need to add try-it-live section to the new page.

**Comment by priyankatuteja** [comment]
@jyaistMap The Pr is ready for review now.

File: guide/01-getting-started/install-and-set-up.ipynb [hunk 1]
```diff
@@ -4,234 +4,100 @@
    "cell_type": "markdown",
```
Comment (rohitgeo) [inline guide/01-getting-started/install-and-set-up.ipynb:4]: <p>This should be normal text, not a heading.</p><p> </p><p>Also, remove the phrase "and iys SDK" from the first sentence.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/764/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (rohitgeo) [inline guide/01-getting-started/install-and-set-up.ipynb:4]: <p>In ReviewNB, this is not showing up as a heading... not sure if thats a ReviewNB bug.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/764/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (rohitgeo) [inline guide/01-getting-started/install-and-set-up.ipynb:4]: <p>"using the steps below"... but there are no steps below?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/764/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Comment (rohitgeo) [inline guide/01-getting-started/install-and-set-up.ipynb:4]: <p>Replace this section with:</p><p> </p><h2>Next Steps</h2><h2><br></h2><p><a href="https://developers.arcgis.com/python/guide/overview-of-the-arcgis-api-for-python/" target="_blank">This overview</a> describes how to use ArcGIS API for Python to write Python code, incorporating capabilities such as mapping, query, analysis, geocoding, routing, portal administration, and more. Browse the <a href="https://developers.arcgis.com/python/sample-notebooks/" target="_blank">sample notebooks</a> to get started. You may choose to experience these sample notebooks in a live environment as they are available as ArcGIS Notebooks.  </p><h2><br></h2>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/764/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

**Comment by priyankatuteja** [comment]
@rohitgeo Thanks for the review. I've updated the notebooks as per your suggestions.

**Review by rohitgeo — APPROVED** [review]
Thanks @priyankatuteja - this should simplify the steps and cleans up the guide a lot.

**Pull request merged**